coral-component-card/src/scripts/Card.js

```diff
@@ -404,8 +404,9 @@ class Card extends BaseComponent(HTMLElement) {
     this.asset = asset;
     
     // In case a lot of alerts are added, they will not overflow the card
-    this.classList.toggle(`${CLASSNAME}--overflow`, this.info.scrollHeight > this.clientHeight);
-  }
+    // Also check whether any alerts are available
+    this.classList.toggle(`${CLASSNAME}--overflow`, this.info.childNodes.length && this.info.scrollHeight > this.clientHeight);
```

Comment:
Your comment says `whether any alerts are available` but you're checking for childNodes which can be textnodes and more. 
I would simply check for `this.info.querySelector('coral-alert, coral-tag')` wdyt ?

Reply:
I would like to take into account text nodes as well, in case the `coral-card-info tag` is used with text as an alert and customized by the user. What do you think?

Reply:
but this also includes html comments